ts/features/services/screens/ServiceDetailsScreen.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect } from "react";
+import { ScrollView, StyleSheet } from "react-native";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import { ServiceId } from "../../../../definitions/backend/ServiceId";
+import { IOStackNavigationRouteProps } from "../../../navigation/params/AppParamsList";
+import { ServicesParamsList } from "../navigation/params";
+import { useIODispatch, useIOSelector } from "../../../store/hooks";
+import { serviceByIdSelector } from "../../../store/reducers/entities/services/servicesById";
+import { loadServiceDetail } from "../../../store/actions/services";
+import { ServicePublic } from "../../../../definitions/backend/ServicePublic";
+import { useHeaderSecondLevel } from "../../../hooks/useHeaderSecondLevel";
+
+export type ServiceDetailsScreenNavigationParams = Readonly<{
+  serviceId: ServiceId;
+  // if true the service should be activated automatically
+  // as soon as the screen is shown (used for custom activation
+  // flows like PN)
+  activate?: boolean;
+}>;
+
+type ServiceDetailsScreenProps = IOStackNavigationRouteProps<
+  ServicesParamsList,
+  "SERVICE_DETAIL"
+>;
+
+const styles = StyleSheet.create({
+  scrollContentContainer: {
+    flexGrow: 1
+  }
+});
+
+export const ServiceDetailsScreen = ({ route }: ServiceDetailsScreenProps) => {
+  const { serviceId } = route.params;
+
+  const dispatch = useIODispatch();
+
+  const servicePot = useIOSelector(state =>
+    serviceByIdSelector(state, serviceId)
+  );
+
+  useEffect(() => {
+    dispatch(loadServiceDetail.request(serviceId));
+  }, [dispatch, serviceId]);
+
+  if (pot.isLoading(servicePot)) {
+    // TODO: add a loading screen
+  }
+
+  return pipe(
+    servicePot,
+    pot.toOption,
+    O.fold(
+      () => (
+        // TODO: add error screen
+        <></>
+      ),
+      service => <ServiceDetailsContent service={service} />
+    )
+  );
+};
```

Comment:
This way you show an error only when your `pot` is `NoneError`, is this the intended behavior? Don't you wanna show a `Toast` or `Alert` even when your `pot` is `SomeError`? What do you think?

Reply:
Good point, yes in this way the `SomeError` is not handled, what do you think about this implementation https://github.com/pagopa/io-app/pull/5607/commits/5cc781d82df5c94acb745e77474465738845d985 ?

Reply:
It depends if you want to show and error UI in both cases or you want to distinguish the case where you have no data and an error, in this case you show the error UI, while if you update the current detail page you only want an Alert or a Toast. What do you think?

Reply:
But I think that you can address al those cases once you add the final UI.

Reply:
After an internal check, we decided to show an error screen in both cases (`noneError` and `someError`)